kbchat/test_config.example.yaml

```diff
@@ -0,0 +1,20 @@
+# Rename this file to `test_config.yaml`
+
+config:
+  bots:
+    alice1:
+      # Alice should have an active Stellar account with a little bit of XLM in it
+      username: "alice"
+      paperkey: "foo bar car..."
+    }
+    alice2:
+      username: "alice"                # should be the same username as alice1 but...
+      paperkey: "yo there paperkey..." # ...this should be a DIFFERENT paperkey than the one for alice1
+    bob1:
+      # Bob should have an active Stellar account with a little bit of XLM in it
+      username: "bob"
+      paperkey: "one two three four..."
+    charlie1:
+      # Charlie should be an account that does not have access to Stellar features
```

Comment:
What do you mean by this comment? Zero balance? Or hasn't accepted the disclaimer?

Reply:
hasn't accepted the disclaimer. reworded to make a little clearer